.goreleaser.yaml

```
@@ -0,0 +1,170 @@
+          cmd: wix extension add -g WixToolset.UI.wixext/5.0.2 WixToolset.Util.wixext/5.0.2
+          output: true
+
+builds:
+    - id: windows-x86_64
+      builder: rust
+      command: build
+      flags:
+          - --release
+          - -p=rioterm
+      targets: [x86_64-pc-windows-msvc]
+
+    - id: windows-arm64
+      builder: rust
+      command: build
+      flags:
+          - --release
+          - -p=rioterm
+      targets: [aarch64-pc-windows-msvc]
+
+    - id: macos
+      builder: rust
+      command: build
+      flags:
+          - --release
+          - -p=rioterm
+      env:
+          - MACOSX_DEPLOYMENT_TARGET={{ if eq .Arch "amd64" }}10.15{{ else }}11.0{{ end }}
+      targets: [x86_64-apple-darwin, aarch64-apple-darwin]
+
+    - id: linux-wayland
+      builder: rust
+      command: build
+      flags:
+          - -p=rioterm
+          - --release
+          - --no-default-features
+          - --features=wayland
+      targets: [x86_64-unknown-linux-gnu, aarch64-unknown-linux-gnu]
+
+    - id: linux-x11
+      builder: rust
+      command: build
+      flags:
+          - -p=rioterm
+          - --release
+          - --no-default-features
+          - --features=x11
+      targets: [x86_64-unknown-linux-gnu, aarch64-unknown-linux-gnu]
+
+release:
+    make_latest: "{{ if .IsNightly }}false{{ else }}true{{ end }}"
+    mode: append
+    prerelease: auto
+    use_existing_draft: false
+    replace_existing_draft: false
+    replace_existing_artifacts: true
+    include_meta: false
+    github:
+        owner: vedantmgoyal9 # raphaelamorim
+        name: rio
+    footer: >-
+        ---
+        Released by [GoReleaser](https://github.com/goreleaser/goreleaser).
+
+nightly:
+    version_template: "{{ incminor .Version }}-nightly"
+    tag_name: nightly
+    publish_release: true
+    keep_single_release: true
+
+# left for @caarlos0, i wasn't able to make it work
+changelog:
+    disable: true
+
+# nfpms:
+#     - formats:
+#           - deb
+#       contents:
+#           - src: ./misc/rio.desktop
+#             dst: /usr/share/applications/rio.desktop
+#           - src: ./misc/rio.terminfo
+#             dst: /usr/share/info/rio.terminfo
+#           - src: ./misc/logo.svg
+#             dst: /usr/share/icons/hicolor/scalable/apps/rio.svg
+
+universal_binaries:
+    - ids: [macos]
+      replace: true
+
+msi:
+    - id: win-x64
+      name: "{{ .ProjectName }}-installer-x86_64"
+      wxs: ./misc/windows/rio.wxs
+      ids: [windows-x86_64]
+      extensions:
+          - WixToolset.UI.wixext
+          - WixToolset.Util.wixext
+      extra_files: [./misc/windows/rio.ico, ./misc/windows/License.rtf]
+    - id: win-arm64
+      name: "{{ .ProjectName }}-installer-aarch64"
+      wxs: ./misc/windows/rio.wxs
+      ids: [windows-arm64]
+      extensions:
+          - WixToolset.UI.wixext
+          - WixToolset.Util.wixext
+      extra_files: [./misc/windows/rio.ico, ./misc/windows/License.rtf]
```

```suggestion
      extra_files:
        - ./misc/windows/rio.ico
        - ./misc/windows/License.rtf
```

I think it was adding the comma to the filename